actionmailbox/test/controllers/ingresses/amazon/inbound_emails_controller_test.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ActionMailbox::Ingresses::AmazonSes::InboundEmailsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    ActionMailbox.ingress = :amazon_ses
+    Rails.application.config.action_mailbox.amazon_ses = ActiveSupport::OrderedOptions.new
+    pem_url = "https://sns.eu-west-1.amazonaws.com/SimpleNotificationService-a86cb10b4e1f29c941702d737128f7b6.pem"
+    stub_request(:get, pem_url).and_return(body: fixture("certificate.pem"))
+  end
+
+  teardown do
+    ActionMailbox.ingress = nil
+    if Kernel.const_defined? :Aws
+      Aws.config[:s3]&.delete(:stub_responses)
+    end
+  end
+
+  test "receiving an inbound email with an s3 action configured" do
+    require "aws-sdk-s3"
```

Comment:
Why is this require here?

Reply:
Required for tests, have moved to `actionmailbox/test/test_helper.rb` underneath `require "webmock/minitest"` - tell me if there is a more sensible place to put this.